feature_engine/discretisation/arbitrary.py

```diff
@@ -107,36 +120,32 @@ def fit(self, X: pd.DataFrame, y: Optional[pd.Series] = None):
         return self
 
     def transform(self, X: pd.DataFrame) -> pd.DataFrame:
-        """Sort the variable values into the intervals.
 
-        Parameters
-        ----------
-        X: pandas dataframe of shape = [n_samples, n_features]
-            The dataframe to be transformed.
-
-        Returns
-        -------
-        X_new: pandas dataframe of shape = [n_samples, n_features]
-            The transformed data with the discrete variables.
-        """
-
-        # check input dataframe and if class was fitted
         X = super().transform(X)
-
-        # transform variables
-        if self.return_boundaries:
-            for feature in self.variables_:
-                X[feature] = pd.cut(X[feature], self.binner_dict_[feature])
-
-        else:
-            for feature in self.variables_:
-                X[feature] = pd.cut(
-                    X[feature], self.binner_dict_[feature], labels=False
+        # check if NaN values were introduced by the discretisation procedure.
+        if X[self.variables_].isnull().sum().sum() > 0:
+
+            # obtain the name(s) of the columns with null values
+            nan_columns = X[self.variables_] \
```

Comment:
I don't think we need the \ to break lines. 

We can write a 1 liner and let black tidy it up.

Reply:
What's black? 

The `\` was how I resolved the `feature_engine/discretisation/arbitrary.py:130:17: E128 continuation line under-indented for visual indent` style error.

Creating a new line w/o the `\` still led to the error. The solution was suggested by PyCharm, which is I realize is not necessarily the style guidebook.

Reply:
Should read like this:
```

            nan_columns = (
                X[self.variables_]
                .columns[X[self.variables].isnull().any()]
                .tolist()
            )
```

with  no \ at the end of the line